geode-core/src/distributedTest/java/org/apache/geode/cache30/PutAllMultiVmDUnitTest.java

```diff
@@ -347,22 +319,12 @@ public void testPutAllFromPreload() throws InterruptedException {
 
   @Test
   public void testSimplePutAll() {
-    Host host = Host.getHost(0);
-    VM vm0 = host.getVM(0);
-    VM vm1 = host.getVM(1);
+    VM vm0 = VM.getVM(0);
+    VM vm1 = VM.getVM(1);
 
-    SerializableRunnable clear = new CacheSerializableRunnable("clear") {
-      @Override
-      public void run2() throws CacheException {
-        try {
-          region.clear();
-        } catch (Exception ex) {
-          ex.printStackTrace();
-        }
-      }
-    };// end of clear
+    // end of clear
 
-    vm0.invoke(new CacheSerializableRunnable("testSimplePutAll1") {
+    vm0.invoke("testSimplePutAll1", new CacheSerializableRunnable() {
```

Comment:
This (and other uses of `CacheSerializableRunnable` below) can be simplified by using a lambda instead:
```
    vm0.invoke("testSimplePutAll1", () -> {
      int cntr = 0, cntr1 = 0;
      for (int i = 1; i < 6; i++) {
        region.put(i, "testSimplePutAll" + i);
        cntr++;
      }
...
    });
```

Reply:
You cannot do that because CacheSerializableRunnable throws a CacheException

`  @Override
  public void run() {
    try {
      run2();
    } catch (CacheException exception) {
      String message = "While invoking \"" + this + "\"";
      throw new CacheSerializableRunnableException(message, exception);
    }
  }
`